vdr/src/client/client.rs

```diff
@@ -263,11 +260,11 @@ pub mod test {
 
     fn contracts() -> Vec<ContractConfig> {
         vec![
-            ContractConfig {
-                address: DID_REGISTRY_ADDRESS.to_string(),
-                spec_path: Some(build_contract_path(DID_REGISTRY_SPEC_PATH)),
-                spec: None,
-            },
+            // ContractConfig {
+            //     address: DID_REGISTRY_ADDRESS.to_string(),
+            //     spec_path: Some(build_contract_path(DID_REGISTRY_SPEC_PATH)),
+            //     spec: None,
+            // },
             ContractConfig {
```

Comment:
If I understand right, and we need to adapt this for did:ethr. Could you please add some TODO comment?